amclient/amclientargs.py

```diff
@@ -52,6 +52,11 @@
     help='Purpose of storage space location valid choices: "AR", "AS", "CP", "DS", "SD", "SS", "BL", "TS", "RP"',
     type=None,
 )
+EVENT_REASON = Arg(
+    name="event_reason", help="Event reason for package deletion request", type=None
+)
+SS_USER_ID = Arg(name="ss_user_id", help="Storage Service user ID", type=None)
```

Comment:
The optional arguments for the command include:

```
  --ss-user-name USERNAME
                        Storage Service username. Default: test
```
which might be a bit confusing with the help text of these positional arguments:

```
  ss_user_id            Storage Service user ID
  ss_user_email         Storage Service user email
```

It might be worth expanding the help text of `SS_USER_ID` and `SS_USER_EMAIL` to explain the deletion request will be on behalf of the user specified in these parameters. Hope this makes sense.